### DISCUSSION TITLE
Language switch back to english

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#225] [General]
I added ?lang=it to the hoe page. HFS displayed in italian.
I clicked on home icon IN HFS then i  refresh the page from the **BRWOSER**  and the HFS is back in english !
actually refreshing any page from the browser would change the language back to english.
I suggest the language be a property of the account and the server adds "lang=CODE" to all pages, in this case refreshing the page from the browser would not change the language
Please Check

**Comment by rejetto** [comment]
You need something like Lang that doesn't reset after refresh?
Please tell me about your use case, why both the automatic language and the forced language you can set in the admin panel doesn't fit your case

**Reply by GeorgeGeorgeGe** [reply]
I am sorry, i did not pay attention to the automatic language selection feature. 
I thought that all pages in English only.
when i read: "Appending ?lang=CODE to address will force a specific language" in the : 
https://github.com/rejetto/hfs#hidden-features 
i thought that is the way to change language in the front-end user.
I am sorry again.

**Comment by rejetto** [comment]
Ok no problem
